pages/shared/studentInstanceQuestion.js

```diff
@@ -0,0 +1,74 @@
+const _ = require('lodash');
+const sqldb = require('@prairielearn/prairielib/sql-db');
+
+const fileStore = require('../../lib/file-store');
+
+module.exports = {
+    async getVariantId(req, res) {
```

Comment:
It's a little unclear to me what this function is doing - it seems to be relying on `callOneRowAsync` to throw if something is invalid? A comment explaining why this is necessary would be good, and maybe an explicit `try`/`catch` with the sproc call would be good too.

Reply:
done